APIs/schemas/edidapi-receiver.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "description": "Describes the EDID Processing API /receivers/{receiverId} base resource",
+  "title": "EDID Processing API /receivers/{receiverId} base resource",
+  "items": {
+    "type": "string",
+    "enum": [
+      "sinks/",
+      "edid/"
```

Comment:
This was moved I beleive

Reply:
I fixed it in 91bd52b080ec3d71f6af658c2a340a64fd8fc09e. Thank you.